msal/tests/Test.MSAL.NET.Unit.net45/CacheV2Tests/InMemoryCachePathStorageTests.cs

```diff
@@ -0,0 +1,389 @@
+﻿// ------------------------------------------------------------------------------
+// 
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+// 
+// This code is licensed under the MIT License.
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+// 
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+// 
+// ------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.CacheV2.Impl.InMemory;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Test.MSAL.NET.Common.Core.Helpers;
+
+namespace Test.MSAL.NET.Unit.net45.CacheV2Tests
+{
+    [TestClass]
+    public class InMemoryCacheKeyStorageTests
+    {
+        private const string FileName = "FileIOTests.bin";
+        private const string TestFolderBase = "FileIOTestsFolder";
+        private byte[] _data;
+        private InMemoryCachePathStorage _io;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            TestCleanup();
```

Comment:
why would this be called first? I suppose we should assume that the tests are properly cleaning up? (aside cleanup is not really doing anything for the moment). Given this is the memory test I assume that as soon as we are out of the scope of the test the GC will handle the cleanup.... however for the file persistence layer I'm assuming proper cleanup is required?

Reply:
Remnants of earlier debugging.  Since TestCleanup is also empty, i've removed TestCleanup and the call here.